spec/unit/changed_rpm_files_helper_spec.rb

```diff
@@ -151,3 +151,12 @@
     )
   end
 end
+
+describe "#get_link_target" do
+  it "returns the link target" do
+    system = "double"
+    expect(system).to receive(:run_command).and_return("/etc/foo\n")
```

Comment:
I think whether the inspector uses `readlink` to get the link target or something else is not of the test's concern. It only couples the test more tightly to the implementation without adding any value imo.